scripts/m3dbnode_bootstrapped.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
This looks better, but I think the issue remains that if their is a placement, but no namespace, the health endpoint will not be available.

@robskillington When you make your changes to turn on the health endpoint sooner, will you be able to make it such that it returns `true` when their is no namespace, but then reverts back to false when the namespace is added and it needs to bootstrap it?

Also, in the case that their is no topology, do we still want to return `true` from the health endpoint? What if their is a topology, but the M3DB node just hasn't received it yet? Seems a little racey.

Thoughts at @robskillington